unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -319,6 +319,8 @@ public void OpenPrivateChat(string userId)
         chatToggleTargetWindow = privateChatWindow;
         chatInputTargetWindow = privateChatWindow;
     }
+
+    public void ShowPrivateChatLoading(bool isActive) { privateChatWindow.SetLoadingMessagesActive(isActive); }
```

Comment:
whats the point of this public method?

Reply:
This is because the current way we have to communicate the `WorldChatWindowController` with the `PrivateChatWindowController` is:
1. `WorldChatWindowController` triggers the `OnRequesPrivateMessages` event.
2. `HUDController` listen to this event and call to `TaskbarHUDController`.
3. `TaskbarHUDController`, that has the reference to the `PrivateChatWindowController`, calls `ShowPrivateChatLoading()`.

Reply:
I think we could solve this dependency, using dependency injection, injecting directly the `PrivateChatWindowController` in the `WorldChatWindowController`, but this would involve a bit refactor because the old code already were following the flow described above. I would attack to this on a separate PR.

Reply:
Forget this. I have removed this method. I was be able do a bit refactor where I'm not longer need to call this from the taskbar 👍 

Done in https://github.com/decentraland/unity-renderer/pull/2484/commits/3353be0a20852adf41295c15a29eabd0f1cf4959